src/props/ProductDetailsProps.tsx

```diff
@@ -0,0 +1,7 @@
+import Product from "../domain/Product"
```

Comment:
Poti lua in calcul si varianta in care iti pui toate tipurile de props intr-un singur fisier 'propsTypes.ts' or smth.